hooks/useInfiniteScroll.tsx

```diff
@@ -27,6 +28,7 @@ export default function useInfiniteScroll({
       setHasNextPage(!result.isLast);
       setPage(1);
       setIsLoading(false);
+      setTotal(result.total && result.total);
```

Comment:
혹시 이렇게 되면 `total`이 없을경우 이상한 값이 들어가진 않나요?!
이 방향성이 더 좋을것 같다고 생각합니다!

```suggestion
    if (result.total) setTotal(result.total);
```